lib/scrape.js

```diff
@@ -155,14 +167,17 @@ async function findOrganization({
 async function findGitHubUser({ display_name }, org) {
   if (!org) return
 
-  display_name = display_name.replace(/ /g, '')
+  const shortName = display_name.replace(/ /g, '')
```

Comment:
spaces are not valid in github usernames, so this is a very basic "convert to valid username".  There was a npm package which checked legality of a name.  What we want is a smarter "convert name into most similar valid github probable username".  maybe a npm package already exists; if not, ... create one ;-)

Reply:
Oh man, alright, I'm running out of package naming ideas ;)